config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root "homepages#index"
+  get "/", to: "homepages#index"
+  resources :works, only: [:index, :show, :new, :create, :edit, :update, :destroy]
+  resources :works do
+    resources :votes, only: [:create]
```

Comment:
This code is a little redundant:

- On line 6, you limit `resources` to generate "only" all 7 of the RESTful routes
- On line 7, you re-generate all 7 routes again

The change here would be to omit line 6.